helm/kube-prometheus/values.yaml

```diff
@@ -1,6 +1,9 @@
 # exporter-node configuration
 deployExporterNode: True
 
+# Grafana
+deployGrafana: False
```

Comment:
I would like to have this the default values as `True`, in that case `kube-prometheus` will provide monitoring, alerting and dashboarding.